routing/pathfind.go

```diff
@@ -666,7 +671,7 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 
 	// Use the nextHop map to unravel the forward path from source to
 	// target.
-	pathEdges := make([]*channeldb.ChannelEdgePolicy, 0, len(next))
+	pathEdges := make([]*channeldb.ChannelEdgePolicy, 0)
```

Comment:
shy this change?

Reply:
*why

I was wondering what removing the cap size achieves.

Reply:
Can just do `var pathedges []*channeldb.ChannelEdgePolicy` perhaps?

Reply:
Oh, the thing here is that `next` doesnt exist anymore, and it's not easy to find out the length of the path with the current structures. Considering a path will probably be a between 1-10 hops, pre-setting the capacity didn't make much sense. I'll update changing the style.